src/pages-and-resources/discussions/app-config-form/apps/shared/AppExternalLinks.jsx

```diff
@@ -24,12 +26,13 @@ function AppExternalLinks({
             <AppConfigFormDivider />
             <h4 className="pt-4">{intl.formatMessage(messages.linkTextHeading)}</h4>
             {linkTypes.map((type) => (
-              <div key={type} className="small text-muted">
```

Comment:
its a shared component, are you sure the removal of this class is not effecting design of components already using this?

Reply:
I' have tested is the change effect on the other design. It is working fine. I made this component more dynamic.